en/Obsidian Sync/Select files and settings to sync.md

```diff
@@ -8,7 +25,8 @@ Any files or settings that have been synced to your [[Local and remote vaults|re
 
 Note that your [[Plans and storage limits|Sync plan]] defines the maximum file size you can sync. With the Standard plan you can sync files up to 5MB, while the Plus plan allows you to sync files up to 200MB.
 
-> [!info] Synced files will not be deleted from your remote vault if you exclude them from sync. If possible, configure the files and settings you want to sync before you start syncing your vault to prevent using storage space unnecessarily.
+> [!info] Files previously synced will remain in your remote vault even if later excluded from sync. Configure the files and settings you wish to sync before initiating the process to avoid unnecessary use of storage space.
```

Comment:
```suggestion
> [!info] Adding a file to the **Excluded files** list will not remove the file from the remote vault if it has already been synced. Configure the files and settings you wish to sync before initiating the process to avoid unnecessary use of storage space.
```

Tried to improve the phrasing here a bit.